packages/static-config/package.json

```diff
@@ -17,6 +17,7 @@
     "dist"
   ],
   "dependencies": {
+    "@swc/core": "1.2.182",
```

Comment:
Is this only being used for "types"? I wonder if we can move to `devDependencies` if that's the case since this package includes a bunch of binaries (I wish there was a `typesDependencies` in package.json 😢).

Reply:
This is huge 🤯 

https://packagephobia.com/result?p=@swc/core@1.2.194

Agreed, it should be moved to `devDependencies`

Reply:
Yes only types are used in `src`. Moved to devDependencies. What a big package! 😮